nifi-nar-bundles/nifi-kerberos-iaa-providers-bundle/nifi-kerberos-iaa-providers-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,37 @@
+nifi-kerberos-iaa-providers-nar
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Spring Framework
+    The following NOTICE information applies:
+      Spring Framework 4.1.6.RELEASE
+      Copyright (c) 2002-2015 Pivotal, Inc.
```

Comment:
Spring Framework is offered under an ASLv2 with no NOTICE (https://github.com/spring-projects/spring-framework). Therefore we should not include it in our notice.